OSCompatibilityLayer.h

```diff
@@ -1,37 +1,35 @@
 #ifndef OS_COMPATABILITY_LAYER_H
 #define OS_COMPATABILITY_LAYER_H
 
-
-
+#include "Log.h"
+#include "iconvlite.h"
 #include <cstdio>
 #include <optional>
-#include <stdint.h>
 #include <set>
+#include <stdint.h>
 #include <string>
-#include "Log.h"
-#include "iconvlite.h"
 
 
 #ifndef MAX_PATH
 #define MAX_PATH 260
-#endif //MAX_PATH
+#endif // MAX_PATH
 
 
 
-//Linux specific defines
+// Linux specific defines
 #ifdef __linux
 
-//sprintf_s is not implemented on Linux. It functions similarly to the function snprintf, but the return values are different.
-//Since return values are not used, they are directly translated for now. If return values will some day be used, the compiler
-//will give an error due to sprintf_s_Linux returning void.
+// sprintf_s is not implemented on Linux. It functions similarly to the function snprintf, but the return values are
+// different. Since return values are not used, they are directly translated for now. If return values will some day be
+// used, the compiler will give an error due to sprintf_s_Linux returning void.
 #define sprintf_s sprintf_s_Linux
-void sprintf_s_Linux (char *__restrict __s, size_t __maxlen, const char *__restrict __format, ...);
+void sprintf_s_Linux(char* __restrict __s, size_t __maxlen, const char* __restrict __format, ...);
```

Comment:
Double-check this, but I don't think sprintf is used anywhere any more. If not, go ahead and strip this out.

Reply:
Not yet. I'm doing all these changes from Window side at the moment, for a full linux cleanup I want to do it from a Linux environment. Only thing I changed so far on LinuxUtils was the common filesystem functions that I know will work on both OS-es. I can't even determine if this is used anywhere as the functions aren't directly referenced and the file isn't included on either of the converters.